test/functional/p2p_feefilter.py

```diff
@@ -111,6 +111,7 @@ def test_feefilter(self):
         txids = [miniwallet.send_self_transfer(fee_rate=Decimal('0.00020000'), from_node=node0)['wtxid'] for _ in range(3)]
         conn.wait_for_invs_to_match(txids)
         conn.clear_invs()
+        self.sync_mempools()  # must be sure node 1 has received all txs
```

Comment:
I am not sure if we still need the same on L102 then?

Also the comment above seems incorrect: There are three transactions being sent, not one.

Reply:
If miniwallet is spending the change output of the previous call (which has been sent on node0), the same tx needs to be sent to node1 for the change output to be spendable there as well.

Generally, we need this everywhere where `from_node` changes between invocations.

Reply:
> Also the comment above seems incorrect: There are three transactions being sent, not one.

Thanks, fixed typo